ga4gh/frontend.py

```diff
@@ -228,6 +274,9 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(SECRET_KEY_LENGTH)
```

Comment:
The pattern would be for whoever is starting the cluster to generate a key, and set it in the startup script, either as an environment variable or an argument. The same key would be used for all servers.